curriculum/js1/7.js

```diff
@@ -5,10 +5,21 @@
  * @returns {boolean}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, primeDivisor = a-1) => {
```

Comment:
```
if (a <= 1 || a%primeDivisor === 0 ) return false;
if (primeDivisor === 1) return true;
return solution(a, primeDivisor - 1);
```

Reply:
I think you need:

if (a<=1 || a%primeDivisor ===0 && primeDivisor > 1) return false:

otherwise,  it will return that 5 is not a prime number because in iteration a(5,1), a false will be returned because 5 divided by 1 returns a 0.